client/dashboard/me/active-subscriptions/active-subscription-expiry.tsx

```diff
@@ -0,0 +1,323 @@
+import { formatCurrency } from '@automattic/number-formatters';
+import { ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { useLocale } from '../../app/locale';
+import { formatDate } from '../../utils/datetime';
+import { SubscriptionBillPeriod } from './constants';
+import {
+	isTemporarySitePurchase,
+	isA4ATemporarySitePurchase,
+	isRecentMonthlyPurchase,
+	getRelativeTimeString,
+	isRenewing,
+	isExpiring,
+	isExpired,
+	isIncludedWithPlan,
+	isOneTimePurchase,
+	isAkismetFreeProduct,
+	isWithinNext,
+	isWithinLast,
+	creditCardHasAlreadyExpired,
+	creditCardExpiresBeforeSubscription,
+} from './util';
+import type { ActiveSubscription } from '../../data/me-active-subscriptions';
+
+export function ActiveSubscriptionExpiry( {
+	purchase,
+	isJetpack,
```

Comment:
Team review: [suggestion] Should we avoid using props such as isJetpack and isDisconnectedSite in favor of passing site as a prop and determined from using utility functions that are shared across HD?